.pre-commit-config.yaml

```diff
@@ -3,14 +3,20 @@ repos:
   rev: v3.2.0
   hooks:
     -   id: end-of-file-fixer
+    -   id: check-toml
 - repo: https://github.com/nbQA-dev/nbQA
-  rev: 0.1.29
+  rev: 0.2.0
```

Comment:
Aren't those the same changes as in #4135 ?

Reply:
Yes - if want to accept #4135 then I can merge upstream/master, I just needed to upgrade the `nbqa` version in order to be able to set configs in `pyproject.toml`

Reply:
Sounds good, just wanted to make sure it was on your radar 👍